.gitignore

```diff
@@ -111,3 +110,146 @@ css/
 TODO.md
 *.lerna-backup
 *.pem
+
```

Comment:
It's all good, but could you move the entry `CUSTOM` to the last of the file?
As it's not generated by the website, but it's from our project-specific settings.
They should be there because they are the items that are most likely to be changed.